server/models/config/index.js

```diff
@@ -0,0 +1,8 @@
+const Airtable = require('airtable');
+require('dotenv').config();
+
+Airtable.configure({
+  endpointUrl: 'https://api.airtable.com',
+  apiKey: process.env.AIRTABLE_API_KEY,
+});
+// const base = Airtable.base(process.env.AIRTABLE_BASE_ID);
```

Comment:
for now let's say you need to define the far nearer base and export it in this file until further notice (I think we'll discuss it on Sunday).

Reply:
OK 
Did I need to remove it for now?

Reply:
@MaiUbeid 
keep it, but just modify it, such as:
```js
const airtable = require('airtable');
require('dotenv').config();

const base = airtable.base(process.env.AIRTABLE_BASE_ID);

module.exports = base;
```
And also, don't forget to make the changes in the parent comment

> * rename `client/src/Components` to `clients/src/components` (lower case `c`)
> * remove `server/controllers/middlewares` (we won't be using `auth` middleware so delete it for now)
> * remove `server/controllers/users`
> * add `server/controllers/maillist`

Reply:
Now I will push these changes